### PR TITLE
Bug 2086459: Continue inspection when some resources are not found

### PR DESCRIPTION
**Pull request by ardaguclu**
Purpose of inspection is to get whatever data it can in best effort.
In case of some resources are not found, inspect still can continue
for other resources.

This PR handles isnotfound error for resources and continues
to inspect. After inspection is completed, it prints error message
to warn user about non-existed resources.

### CODE REVIEW

File: pkg/cli/admin/inspect/inspect.go [hunk 1]
```diff
@@ -10,6 +10,8 @@ import (
 	"strings"
 	"time"
 
+	apierrors "k8s.io/apimachinery/pkg/api/errors"
```
Comment (soltysh) [inline pkg/cli/admin/inspect/inspect.go:13]: Nit: I like those imports nicely grouped into the following groups:
1. golang imports
2. 3-rd party
3. k8s imports
4. openshift imports

**Review by soltysh — APPROVED** [review]
/approve

**Review by soltysh — APPROVED** [review]
/lgtm

**Comment by ardaguclu** [comment]
/hold

Will add bugzilla id.

**Pull request merged**